### ISSUE TITLE
Scrcpy and Powershell applications crashing due to presence of scrcpy files

### ISSUE THREAD

**Issue by EpicViv** [Genymobile/scrcpy#1967]
Hello. 

I have been using scrcpy for a few months now and I wanted to refresh my platform tools and scrcpy files to stay updated with the latest update. However, after doing this, the scrcpy.exe window will not stay open. After launching it closes after 2-3 seconds consistently. When I get rid of the scrcpy files and restart my computer a few times, the Powershell window works at intended. 

Is there something I am doing wrong or any fix to this?

**Comment by rom1v** [comment]
What is the output in the console when it closes? (launch it from `cmd.exe`)

**Comment by EpicViv** [comment]
There is no output. I have attached a screenshot of the terminal. Everything simply runs as if scrcpy is still running. When I turn the screen off using Alt + O, the phone's screen continues to stay off even when the window closes.

![Terminal Screenshot](https://user-images.githubusercontent.com/70602891/101925105-1bf3e000-3b97-11eb-8488-41e6b105e7e9.jpg)

**Comment by rom1v** [comment]
When you took this screnshot of your terminal, was the window open? When it closed, the terminal did not contain more output? At least, it should give you the console prompt.

**Comment by EpicViv** [comment]
This screenshot is after the window closed. There is no output by the console or anything.

**Comment by rom1v** [comment]
Hmmm, that's very weird.

Which terminal is it? (PowerShell? cmd.exe?) Could you test within `cmd.exe`, please?

Also, what is the output of `adb logcat` when it closes:

```bash
adb logcat -c  # clear logs
scrcpy
adb logcat -d > log.txt
```

**Comment by EpicViv** [comment]
Hello. 

This comment may make me seem naive and inexperienced, but I was able to determine that my previous theory is false. The presence of the scrcpy files is **not** causing the crashing of the scrcpy window. Rather, it was Chrome Dev. I know this because the crashing had spread to other apps, such as a few other games and even the Chrome browser itself. This may have been due to some instability with Chrome Dev. I uninstalled Chrome Dev and install the normal Chrome browser and everything seems to be working as expected as of now. Sorry for any confusion or worry this may have caused.
